modules/ROOT/pages/platform/api/authentication.adoc

```diff
@@ -86,6 +91,33 @@ curl --request POST 'https://api.neo4j.io/oauth/token' \
 
 <1> The `--user` option sets the `Authorization` header for you, handling the base64 encoding of the client ID and client secret.
 
+=====
+[.include-with-Python]
+=====
+
+[source, python]
+----
+import requests
+from requests.auth import HTTPBasicAuth
+
+# Make the request to the Aura API Auth endpoint
+response = requests.request(
+    "POST",
+    "https://api.neo4j.io/oauth/token",
+    headers={"Content-Type": "application/x-www-form-urlencoded"},
+    data={"grant_type": "client_credentials"},
+    auth=HTTPBasicAuth(client_id, client_secret),
+    timeout=10
+)
+
+# Obtain the token to use with our Aura API from the response
+token = response.json()["access_token"]
```

Comment:
```suggestion
print(response.json())
```

I would say let's just print the whole response in JSON format, so we are consistent with the output from cURL.